### ISSUE TITLE
Malay translation in Surah Luqman, verse 20

### ISSUE THREAD

**Issue by Qataada** [quran/quran.com-frontend#1182]
I don't know Malay but I found this message in https://quran.zendesk.com/hc/en-us/community/posts/360028189612-Malay-translation-in-Lukman-20

"Lukman: 20
ABDULLAH MUHAMMAD BASMEIH
Tidakkah kamu memperhatikan bahawa Allah telah memudahkan untuk kegunaan kamu apa yang ada di langit dan yang ada di bumi, dan telah melimpahkan kepada kami (Kamu) nikmat-nimatNya yang zahir dan yang batin? Dalam pada itu, ada di antara manusia orang yang membantah mengenai (sifat-sifat) Allah dengan tidak berdasarkan sebarang pengetahuan atau sebarang petunjuk; dan tidak juga berdasarkan mana-mana Kitab Allah yang menerangi kebenaran.
 

the word kami should be kamu.."

**Comment by athdromeda** [comment]
I agree. I'm Indonesian. Our language similar to Malay.  _'alaikum'_ means _'kepada kamu'._

**Comment by mafifiakamil** [comment]
Assalamualaikum,

Agreed with @athdromeda , am from Malaysia. The correction is valid, it should be "kamu" instead of "kami". Please let me know where I can help to correct this in the code.

Thanks
